.travis.yml

```diff
@@ -73,8 +73,17 @@ deploy:
   on:
     tags: true
     repo: kubevirt/kubevirt
+- provider: pages
+  skip_cleanup: true
+  repo: https://github.com/kubevirt-incubator/api-reference.git
```

Comment:
I think it should be `kubevirt-incubator/api-reference.git`

because there is `github_url` field

Reply:
I think that also the `.git` is too much.